### PR TITLE
enable log operator jobs

### PR DESCRIPTION
**Pull request by jcantrill**
This PR creates jobs for:
* cluster-logging-operator
* elasticsearch-operator

@stevekuznetsov I presume this enables 'compile/build' to block merges and that I can further update config to enable 'test' with the various test blocks?

### CODE REVIEW

File: ci-operator/config/openshift/cluster-logging-operator/master.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+test_base_image:
```
Comment (droslean) [inline ci-operator/config/openshift/cluster-logging-operator/master.yaml]: `test_base_image` is deprecated, use `build_root.image_stream_tag` instead
https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#build_root

File: ci-operator/config/openshift/elasticsearch-operator/master.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
+  tag_overrides: {}
+test_base_image:
```
Comment (droslean) [inline ci-operator/config/openshift/elasticsearch-operator/master.yaml]: `test_base_image` is deprecated, use `build_root.image_stream_tag` instead
https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md#build_root

**Comment by jcantrill** [comment]
re-updated...

File: ci-operator/config/openshift/elasticsearch-operator/master.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+images:
+- from: base
+  to: origin-elasticsearch-logging-operator
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/elasticsearch-operator/master.yaml]: Is your Dockerfile in the root of your repo?

**Comment by stevekuznetsov** [comment]
You can try to test this locally by logging into api.ci.openshift.org and then running:

```
docker run -it -e KUBECONFIG=/kube.config -v “${HOME}/.kube/config”:/kube.config -v “$(pwd)/ci-operator/config/openshift/cluster-logging-operator”:/config registry.svc.ci.openshift.org/ci/ci-operator:latest --config /config/master.yaml --target [images] --git-ref ”openshift/cluster-logging-operator@master”
```

**Comment by stevekuznetsov** [comment]
@jcantrill  can you pull down the generator again and re-generate? We updated some stuff. Otherwise this LGTM

**Comment by jcantrill** [comment]
@stevekuznetsov rebased

**Comment by stevekuznetsov** [comment]
@jcantrill can you pull down the generator again and re-generate? We updated some stuff again. Ping me on Slack once you do it and we can merge quickly

**Pull request merged**